lib/utils.js

```diff
@@ -18,10 +18,11 @@ export const decodeUtf8Base64 = ifDefined((input) => decodeUtf8(decodeBase64(inp
 
 export const binaryStringToArray = (args) => openpgp.util.str_to_Uint8Array(args);
 export const arrayToBinaryString = (args) => openpgp.util.Uint8Array_to_str(args); // eslint-disable-line new-cap
+export const arrayToHexString = (args) => openpgp.util.Uint8Array_to_hex(args); // eslint-disable-line new-cap
 export const concatArrays = (args) => openpgp.util.concatUint8Array(args);
 
-export const getHashedPassword = (password) =>
-    localBtoa(arrayToBinaryString(openpgp.crypto.hash.sha512(binaryStringToArray(password))));
+export const sha512 = (args) => openpgp.crypto.hash.sha512(args);
+export const md5 = (args) => openpgp.crypto.hash.md5(args);
```

Comment:
Can you use aliases ?

Reply:
No because of the way openpgp is dynamically set

Reply:
Can you explain ? I don't understand

Reply:
In order for openpgp to become available for this library it needs to be set by the user of pmcrypto using `pmcrypto.init(openpgp)`. It's like that because the version of openpgp is dynamically loaded and some parts (requiring pmcrypto) of the application can be loaded before openpgp is loaded. So then using an alias would fail because it does not have openpgp yet.